components/eamxx/cmake/machine-files/crusher.cmake

```diff
@@ -1,7 +0,0 @@
-# Load all kokkos settings from Ekat's mach file
-set (EKAT_MACH_FILES_PATH ${CMAKE_CURRENT_LIST_DIR}/../../../../externals/ekat/cmake/machine-files)
```

Comment:
Why are we removing this file?

Reply:
There is no crusher machine. we have crusher-gpu and crusher-cpu machines.